samples/simple_vec_mul/simple_mul_int.c

```diff
@@ -38,27 +41,73 @@ iree_status_t Run() {
   iree_vm_instance_t* instance = NULL;
   IREE_RETURN_IF_ERROR(
       iree_vm_instance_create(iree_allocator_system(), &instance));
+
+#if defined(BUILD_INLINE_HAL)
```

Comment:
Do we want to add a configuration for this code path later?

Reply:
Agreed. Have factored this our to be pushed later.